src/app/modules/users/add-user/add-user.component.scss

```diff
@@ -32,3 +32,12 @@
 .fx-spacer {
     flex: 1 1 auto;
 }
+.spin {
+color: $primary-color;
+}
+
+.mat-spinner {
```

Comment:
USe the theme feature in the angular material or override using a class. Please don't modify the library classes

Reply:
applied